src/containers/PaymentInfo/PaymentInfo.tsx

```diff
@@ -58,17 +64,19 @@ const PaymentInfo = ({
         </div>
       ) : (
         <>
-          <section>
-            <SectionHeader>
-              <>
-                {t(
-                  'paymentinfo.current-payment-method',
-                  'Current payment method'
-                )}
-              </>
-            </SectionHeader>
-            <PaymentMethod />
-          </section>
+          {!isEditDeliveryDetailsPopupOpened && (
+            <section>
+              <SectionHeader>
+                <>
+                  {t(
+                    'paymentinfo.current-payment-method',
+                    'Current payment method'
+                  )}
+                </>
+              </SectionHeader>
+              <PaymentMethod />
+            </section>
+          )}
```

Comment:
How does it look? Could you add screenshots to the pull request description? Shouldn't we hide the entire section (with payment history) when this popup is open?

Reply:
How it looks now:
<img width="1028" alt="image" src="https://github.com/Cleeng/mediastore-sdk/assets/68298935/814c2020-5039-4e0d-9778-a79290e8abdb">

Reply:
And also add title and text to popup first step

Reply:
I'm still not sure why we need to embed Transactions when `isEditDeliveryDetailsPopupOpened`. Any particular reason?

Reply:
This Popup is displayed inside `Transactions` component, as I understand we have `TransactionList` component that can be used alone without `PaymentInfo` so this Popup needs to be deeper. Instead of returning solo `Transactions` here in `PaymentInfo` I can return only `EditDeliveryDetailsPopup` but it will do same thing.

Reply:
Now return only `EditDeliveryDetailsPopup` and made it a bit simplified